web/toDoList.jsp

```diff
@@ -1,160 +1,147 @@
-                        },
-                        method: 'POST'
-                    }).done(function (data) {
-                        idOfLastItem = parseInt(data);
-                        if (listEmpty) {
-                            listEmpty = false;
-                            $('#listBody').prepend("<div class='toDoListDataStyle' id='listItemId-" + idOfLastItem + "'><span contenteditable='true' id='item-" + idOfLastItem + "' class ='itemStyle'>" + itemName + " </span><span class='checkboxStyle' id='checkboxId-" + idOfLastItem + "' ><input type='checkbox'/></span><span id='remainderButton-" + idOfLastItem + "' class='button btn btn-default btn-sm remainderButtonStyle'>Reminder</span></div>");
-                            $('#checkboxId-' + idOfLastItem).css('visibility', 'hidden');
-                        } else {
-                            $('#listBody').last().append("<div class='toDoListDataStyle' id='listItemId-" + idOfLastItem + "'> <span contenteditable='true' id='item-" + idOfLastItem + "' class ='itemStyle'>" + itemName + " </span><span class='checkboxStyle' id='checkboxId-" + idOfLastItem + "' ><input type='checkbox'/></span><span id='remainderButton-" + idOfLastItem + "' class='button btn btn-default btn-sm remainderButtonStyle'>Reminder</span></div>");
-                            $('#checkboxId-' + idOfLastItem).css('visibility', 'hidden');
-                        }
-                        setUpClickListeners("#listItemId-" + idOfLastItem);
-                        updateItemFunction("#item-" + idOfLastItem);
-                        reminderFunction("#remainderButton-" + idOfLastItem);
-                    }).fail(function () {
-                    });
-                    itemInputNode.val("");
-                }
-            });
-            var setUpClickListeners = function (Id) {
-                mouseEventFunction(Id);
-                deleteFunction(Id);
-                if (($('#listBody').children().length) == 0) {
-                    listEmpty = true;
-                    $('#emptyListMessage').css("display", "block");
+</div>
+<script>
+    $(function () {
+        $('.datetimepicker').datetimepicker({});
+        var timeLeftMessageFunction = function (id) {
+            $(id).click(function () {
+                var resetPanel = $("#resetPanel");
+                resetPanel.insertAfter(this);
+                var timeLeft = $(this).html();
+                $("#message").html("time remaining: " +timeLeft.split(":").pop());
+                resetPanel.slideToggle();
+                $("#editButton").click(function () {
+                })
+            })
+        };
+        $(".timeLeftMessageStyle").each(function () {
+            var idOfTimeLeftMessage = '#' + this.id;
+            timeLeftMessageFunction(idOfTimeLeftMessage);
+        });
+        $(".remainderButtonStyle").each(function () {
+            var idOfRemainder = '#' + this.id;
+            reminderFunction(idOfRemainder);
```

Comment:
also, make `setupReminderButton` and `setupTimeLeftButton` to accept jquery object instead of id. that way, you can directly pass `$(this)`. instead of doing 
```javascript
var idOfRemainder = '#' + this.id;
reminderFunction(idOfRemainder);
```